storage/innobase/handler/ha_innodb.cc

```diff
@@ -21841,7 +21841,7 @@ static void update_innodb_redo_log_encrypt(THD *thd, SYS_VAR *var,
 
   if (srv_redo_log_encrypt != REDO_LOG_ENCRYPT_OFF &&
       srv_redo_log_encrypt != target) {
-    push_warning_printf(thd, Sql_condition::SL_WARNING, ER_WRONG_ARGUMENTS,
+    push_warning_printf(thd, Sql_condition::SL_WARNING, ER_REDO_ENCRYPTION_CANT_BE_CHANGED,
```

Comment:
Error not warning should be returned?

Reply:
I kept it as a warning, as other similar messages (for example, if the user tries to turn on encryption on a read only server) are warnings too, and those are upstream.